mne/viz/raw.py

```diff
@@ -178,6 +173,17 @@ def plot_raw(raw, events=None, duration=10.0, start=0.0, n_channels=20,
         then excessive values are not shown, creating gaps in the traces.
     show_first_samp : bool
         If True, show time axis relative to the ``raw.first_samp``.
+    group_by : str
+        How to group channels. 'type' groups by channel type, 'original'
+        plots in the order of ch_names, 'selection' uses Elekta's channel
+        groupings (only works for Neuromag data), 'position' groups the
+        channels by the positions of the sensors. 'selection' and 'position'
+        modes allow butterfly mode (press 'b') and custom selections by using
+        lasso selector on the topomap. Pressing ``ctrl`` key while selecting
+        allows appending to the current selection. Channels marked as bad
+        appear with red edges on the topomap. 'butterfly' is equal to
+        'position', but the plotter is started in butterfly mode. 'type' and
+        'original' groups the channels by type in butterfly mode.
```

Comment:
There is no way to start `butterfly` mode with `'position'`.

It seems like we're trying to force a `butterfly` param into the `group_by` when it should really be separate.

Both `order` and `group_by` affect channel order currently. Why not leave `order` allowing string_types and thus the only thing that sets channel order, and then also allow `butterfly=True | False`? Seems cleaner to me.